src/e2e/pages/dashBoardPage.ts

```diff
@@ -155,6 +167,7 @@ export class DashboardPage {
     this.servicesFirefoxMobile = page.getByRole("link", {
       name: "⁨Firefox⁩ for Mobile",
     });
+    this.servicesMozilla = page.locator('[data-key="mozilla"] > a');
```

Comment:
OK, I think this is the bottom link in the DOM of the popup bento box menu:
```html
<li role="menuitem" tabindex="-1"
     data-key="mozilla" class="AppPicker_menuItemWrapper__TpxV9 ">
  <a href="https://www.mozilla.org/?utm_source=monitor.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=bento&amp;utm_content=desktop"
     class="AppPicker_menuLink__O8tJX AppPicker_mozillaLink__LTe4E"
     target="_blank" rel="noopener noreferrer">Made by ⁨Mozilla⁩</a></li>
```

Wonder if something like `page.getByRole("listitem", { name: "Made by Mozilla" })` might work (possibly not since I see you're using the `>` child selector).

Reply:
Ok! I will chain locators, which should achieve the same behavior

Reply:
nvm, it has these weird invisible characters here too, and the locator chain becomes rather long... I can change it to be a locator chain, but imo the argument in .locator makes it cleaner.